models/workitemtype_test.go

```diff
@@ -63,27 +56,22 @@ func TestMarshalEnumType(t *testing.T) {
 		Required: true,
 	}
 
-	timeNow := time.Now()
-	wt := WorkItemType{
-		Lifecycle: Lifecycle{
```

Comment:
yeah, that's what I implied. :)  This is a good test case since it's the zero values that had hit us hard.